nilearn/_utils/extmath.py

```diff
@@ -75,3 +78,45 @@ def is_spd(M, decimal=15, verbose=1):
     if not ispd and verbose > 0:
         print("matrix has a negative eigenvalue: %.3f" % eigvalsh.min())
     return ispd
+
+
+def check_threshold(threshold, data, percentile_calculate, name):
```

Comment:
I think it should be because this function also includes to give threshold based on ```scoreatpercentile```.